### ISSUE TITLE
Integration with audit-ci or similar

### ISSUE THREAD

**Issue by kevinfairclough** [tnobody/lerna-audit#33]
https://www.npmjs.com/package/audit-ci

Would it be possible to integrate this with the above, or similar package to integrate lerna-audit with CI pipelines?

**Comment by svettwer** [comment]
Hi :wave: 

You can easily use learna-audit in CI pipelines without any further integration. There is not a dedicated GitHub or something but whenever you're able to  run npm commands, you're ready to go.

Example:
```json
{
  "name": "example",
  "scripts": {
    "audit": "npm audit fix && lerna-audit"
  },
  "devDependencies": {
    "lerna": "4.0.0",
    "lerna-audit": "1.3.3"
  }
}
```

`npx run audit` would perform an audit on the root package and an audit on the learna packages. This snippet could easily be used in any CI Platform.

TL;DR: No integration required => learna-audit can be used in CI out of the box.

BR,
Sven

**Comment by svettwer** [comment]
@kevinfairclough Did this answer your question? =)
